sdk/communication/Azure.Communication.PhoneNumbers/src/SipRouting/SipRoutingClient.cs

```diff
@@ -130,18 +130,84 @@ public virtual Response<SipTrunk> GetTrunk(
         /// Get <see cref="SipTrunk"/> with provided FQDN.
```

Comment:
Feedback from Review:
We should also expose "Domains", so we should implement these methods:

    getDomain(fqdn: string), returns SipDomain
    getDomainAsync(fqdn: string,  CancellationToken cancellationToken = default), returns Task<SipDomain>
    getDomains(), returns Response<IReadOnlyList<SipDomain>>
    getDomainsAsync(CancellationToken cancellationToken = default), returns Response<IReadOnlyList<SipDomain>>
    setDomain(domain: SipDomain, options?: OperationOptions): Promise<SipDomain>;
    setDomains(domains: SipDomain[], options?: OperationOptions): Promise<SipDomain[]>;
    
The internal logic is similar to those handling trunks.
Also, we need to expose SipDomain object that would be handled similarly as SipTrunk.